CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Arguments of an internal command are not parsed if it is forced over its existent
   external counterpart.
 
+- aeon: Fix SSL paths configuration for aeon connection.
```

Comment:
Please, remove extra newline BEFORE the entry.

Reply:
Done